src/patternfly/components/Select/select-checkbox-groups.hbs

```diff
@@ -4,44 +4,48 @@
     {{{select-multi-attribute}}}
   {{/if}}>
   {{#> form}}
-    {{#> form-group form-group--modifier="pf-c-form__fieldset" form-group--attribute=(concat 'aria-labelledby="' id '-label"')}}
+    {{#> form-group form-group--attribute=(concat 'aria-labelledby="' id '-label"')}}
       <div class="pf-c-select__menu-group">
-        <div class="pf-c-select__menu-group-title">Status</div>
-        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-running"') check--modifier="pf-c-select__menu-item"}}
-          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-running" name="running"')}}{{/check-input}}
-          {{#> check-label check-label--type="span"}}Running{{/check-label}}
-        {{/check}}
-        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-stopped"') check--modifier="pf-c-select__menu-item"}}
-          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-stopped" name="stopped"')}}{{/check-input}}
-          {{#> check-label check-label--type="span"}}Stopped{{/check-label}}
-        {{/check}}
-        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-down"') check--modifier="pf-c-select__menu-item"}}
-          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-down" name="down"')}}{{/check-input}}
-          {{#> check-label check-label--type="span"}}Down{{/check-label}}
-        {{/check}}
-        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-degraded"') check--modifier="pf-c-select__menu-item"}}
-          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-degraded" name="degraded"')}}{{/check-input}}
-          {{#> check-label check-label--type="span"}}Degraded{{/check-label}}
-        {{/check}}
-        {{#> check  check--type="label" check--attribute=(concat 'for="' id '-needsMaintenance"') check--modifier="pf-c-select__menu-item"}}
-          {{#> check-input check-input--attribute=(concat 'type="checkbox" id="' id '-needsMaintenance" name="needsMaintenance"')}}{{/check-input}}
-          {{#> check-label check-label--type="span"}}Needs Maintenance{{/check-label}}
-        {{/check}}
+        <div aria-hidden="true" id="group-status" class="pf-c-select__menu-group-title">Status</div>
```

Comment:
can you make `id="group-status"` unique so it doesn't conflict with the other example? Maybe use `{{id}}` to generate the se ID's and the `aria-labelledby` values?

Reply:
Yes!